ui/components/multichain/account-details/account-details-key.js

```diff
@@ -45,7 +45,11 @@ export const AccountDetailsKey = ({ accountName, onClose, privateKey }) => {
         padding={4}
         gap={4}
       >
-        <Text variant={TextVariant.bodySm} style={{ wordBreak: 'break-word' }}>
+        <Text
+          data-testid="account-details-key"
+          variant={TextVariant.bodySm}
+          style={{ wordBreak: 'break-word' }}
```

Comment:
I think we can use the `overflowWrap` property here instead of inline style
```suggestion
          overflowWrap={OverflowWrap.BreakWord}
```

Reply:
Just tested and that does *not* work -- the text breaks well out of the box.